sql/updates/world/3.3.5/XXXX_XX_XX_XX_world.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_hun_pet_cower';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (1742, 'spell_hun_pet_cower');
```

Comment:
TC codestyle recommends starting SQL files with a comment line, in case the preceding SQL file is missing a NewLine at EOF.
```suggestion
-- 
DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_hun_pet_cower';
INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (1742, 'spell_hun_pet_cower');
```
(if there is a practical need, you could choose to add text in the top comment line)